src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Värdet på nätmasken bör vara mellan {{ min }} och {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Filnamnet är för långt. Det bör ha {{ filename_max_length }} tecken eller mindre.|Filnamnet är för långt. Det bör ha {{ filename_max_length }} tecken eller färre.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Styrkan på lösenordet är för låg. Använd ett starkare lösenord.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Denna värde innehåller tecken som inte är tillåtna enligt den aktuella begränsningsnivån.</target>
```

Comment:
If I read this message, I would have no idea what it meant. 😕 

Would this be better?
```suggestion
                <target>Denna värde innehåller tecken som inte är tillåtna.</target>
```